custom_components/hacs/helpers/classes/repository.py

```diff
@@ -412,6 +414,10 @@ async def remove_local_directory(self):
 
                 while os.path.exists(local_path):
                     await sleep(1)
+            else:
+                self.logger.debug(
+                    f"Presumed local content path {local_path} does not exist"
```

Comment:
It may exist, but may not be valid for removal

Reply:
The check in the `if` condition is purely an existence check (`os.path.exists()`). How does the validity come into play here?

Reply:
It does not, I thought it was an else to `if not is_safe_to_remove(local_path):`, my bad.